ts/types.ts

```diff
@@ -32,5 +32,6 @@ export type TestResult = {
 
 export type CompilerOpts = {
   consoleLog: (message: string) => void,
-  version: () => void
-}
\ No newline at end of file
+  version: () => void,
+  whitelistCode?: { [key: string]: string }
```

Comment:
I suggest this not be optional. Let's force the user to explicitly supply an empty object.